dist/index.js

```diff
@@ -4,6 +4,7 @@ import path from "path";
 import esbuild from "esbuild";
 import { parseArgs } from "./args.js";
 import { buildOptions, watchOptions } from "./esbuild.js";
+const cloneDeep = require("lodash.clonedeep");
```

Comment:
```suggestion
import cloneDeep from "lodash.clonedeep";
```

fixes:

```
const cloneDeep = require("lodash.clonedeep");
                  ^

ReferenceError: require is not defined in ES module scope, you can use import instead
This file is being treated as an ES module because it has a '.js' file extension and 'package.json' contains "type": "module". To treat it as a CommonJS script, rename it to use the '.cjs' file extension.
```

Reply:
My changes: https://github.com/hanami/assets-js/compare/fix/prefix-static-assets-with-slice-name...parndt:assets-js:fix/prefix-static-assets-with-slice-name